CHANGELOG.md

```diff
@@ -4,7 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 ## [1.1.4.1](https://github.com/unzerdev/php-sdk/compare/1.1.4.0..1.1.4.1)
 ### Added
-*   Added Apple Pay example.
+* Added Apple Pay example.
+
+### Change
+* Adjust `cancelAmount` logic to work properly with Invoice Secured payments.
 
 ### Updated
```

Comment:
@Ryouzanpaku There is no "Updated" - keyword, please check "how to keep a changelog" for valid captions.
This change belongs to the "Changed" section.

Reply:
true, its "updated" now.